src/time_unit/minutes.rs

```diff
@@ -19,6 +28,12 @@ impl TimeUnitField for Minutes {
         59
     }
     fn ordinals(&self) -> &OrdinalSet {
-        &self.0
+        match &self.ordinals {
+            Some(ordinal_set) => &ordinal_set,
+            None => &ALL
+        }
+    }
+    fn is_specified(&self) -> bool {
+        self.ordinals.is_some()
```

Comment:
If someone wrote `0-23` for the hours field, that would be equivalent to `*` but this would return false. Is that the intended effect? What if this were something like:
```suggestion
        let max_supported_ordinals = (Self::inclusive_max() - Self::inclusive_min()) + 1;
        let is_all = self.ordinals
            .map(|o| o.len() == max_supported_ordinals)
            .unwrap_or(true);
        !is_all
```
This could be a default impl shared across all `TimeUnitField` types.

Reply:
(If this were renamed from `is_specified` to `is_all` we'd obviously drop the `!` in the returned value.)

Reply:
Thios could be an option. I feel like is_all is still a really weird function to call on years though

Reply:
If I'd make that change, I would rather write it as:
```rust
fn is_all(&self) -> bool {
    let max_supported_ordinals = Self::inclusive_max() - Self::inclusive_min() + 1
    self.ordinals.map_or(true, |o| o.len() == max_supported_ordinals))
}
```

Reply:
Yep, that's better. 👍 Didn't know about `map_or`.

Reply:
>Didn't know about map_or.
Me neither till I tried running rustfix --pedantic. Might not be a bad idea to run that after this PR cycle.

I can do this, but ordinals() in TimeUnitField doesn't return an Option so I just changed it to 
```rust
    fn is_all(&self) -> bool {
        let max_supported_ordinals = Self::inclusive_max() - Self::inclusive_min() + 1;
        self.ordinals().len() == max_supported_ordinals as usize
    }
```
Which I'm sure is acceptable as well